verify/index.ts

```diff
@@ -11,22 +11,18 @@ async function verify() {
         const target_artifact_ref = core.getInput('target_artifact_reference');
         const trust_policy = core.getInput('trust_policy'); // .github/trustpolicy/trustpolicy.json
         const trust_store = core.getInput('trust_store'); // .github/truststore
-        let output;
         // configure Notation trust policys
-        execSync(`notation policy import ${trust_policy}`);
-        output = execSync(`notation policy show`, { encoding: 'utf-8' });
-        console.log(output);
+        await exec.getExecOutput(`notation policy import ${trust_policy}`);
+        await exec.getExecOutput(`notation policy show`);
         // configure Notation trust store
-        configTrustStore(trust_store);
-        output = execSync(`notation cert ls`, { encoding: 'utf-8' });
-        console.log(output);
+        await configTrustStore(trust_store);
+        await exec.getExecOutput(`notation cert ls`);
         // verify core logic
         if (process.env.NOTATION_EXPERIMENTAL) {
-            output = execSync(`notation verify --allow-referrers-api ${target_artifact_ref} -v`, { encoding: 'utf-8' });
+            await exec.getExecOutput(`notation verify --allow-referrers-api ${target_artifact_ref} -v`);
         } else {
-            output = execSync(`notation verify ${target_artifact_ref} -v`, { encoding: 'utf-8' });
+            await exec.getExecOutput(`notation verify ${target_artifact_ref} -v`);
```

Comment:
What will happen if verification fail? Is `exitCode` checked? What about the output?

Reply:
Discussed offline. The `error` will be caught automatically and the workflow would fail accordingly. For example: https://github.com/notation-playground/notation-integration-with-ACR-and-AKV/actions/runs/5584172975/jobs/10205644499